llm/qwen_cpp/README.md

```diff
@@ -0,0 +1,57 @@
+# LLM
+
+This application showcases inference of a large language model (LLM). It doesn't have much of configuration options to encourage the reader to explore and modify the source code. There's a Jupyter notebook which corresponds to this pipeline and discusses how to create an LLM-powered Chatbot: https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/254-llm-chatbot.
+
+> Note  
+This pipeline is not for production use.
+
+## How it works
+
+The program loads a tokenizer and a model (`.xml` and `.bin`) to OpenVINO™. The model is reshaped to batch 1 and variable prompt length. A prompt is tokenized and passed to the model. The model greedily generates token by token until the special end of sequence (EOS) token is obtained. The predicted tokens are converted to chars and printed in a streaming fashion.
+
+## Supported models
+
+1. Qwen
+    1. https://huggingface.co/Qwen/Qwen-7B
+    2. https://huggingface.co/Qwen/Qwen-7B-Chat
+    3. https://huggingface.co/Qwen/Qwen-7B-Chat-Int4
+
+## Qwen Demo
+### Step 1: Download and convert the model
+
+```sh
+cd openvino.genai/llm_bench/python
+python -m pip install -r requirements.txt
+python convert.py --model_id Qwen/Qwen-7B-Chat-Int4 --output_dir Qwen-7B-Chat-GPTQ_INT4_FP16-2K --precision FP16
```

Comment:
Is there a way to convert without cuda?

For CPU PyTorch I get `AssertionError: Torch not compiled with CUDA enabled`, and for full PyTorch version I get `RuntimeError: Found no NVIDIA driver on your system.`

Reply:
Yes, pls check: https://github.com/openvinotoolkit/openvino.genai/blob/master/llm_bench/python/NOTES.md#qwen-7b-chat-int4---torch-not-com